web/package.json

```diff
@@ -29,8 +29,8 @@
     "@types/react-redux": "^7.1.16",
     "@types/react-router-dom": "^5.3.0",
     "@types/webpack-env": "^1.15.1",
-    "@typescript-eslint/parser": "^5.0.0",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
```

Comment:
Why this change?

Reply:
I think at some point I ran a yarn update and this happened. I don't remember what I was doing at the time. I'll revert this.

Reply:
oh it might have been a dependency that was installed when I initially installed the `copy-to-clipboard` package